sdk/textanalytics/Azure.AI.TextAnalytics.Protocol/demo/Program.cs

```diff
@@ -0,0 +1,90 @@
+﻿using Azure;
+using Azure.AI.TextAnalytics.Protocol;
+using Azure.Core;
+using System;
+
+namespace DemoApp
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            TextAnalyticsClient client = new TextAnalyticsClient(new Uri("<fill-me-in>"), new AzureKeyCredential("<fill-me-in>"));
+
+            // Passing the body "in-line" and using the object constructor with an anonymous object.
+            {
+                DynamicResponse res = client.GetLanguages(new JsonData(new
+                {
+                    doucments = new JsonData(new object[]
+                    {
+                        new
+                        {
+                            countryHint = "US",
+                            id = 1,
+                            text = "Hello world",
+                        },
+                        new
+                        {
+                            id = 2,
+                            text = "Bonjour tout le monde"
+                        },
+                        new
+                        {
+                            id = 3,
+                            text = "La carretera estaba atascada. Había mucho tráfico el día de ayer."
+                        }
+                    })
+                }));
+
+                Console.WriteLine($"Status: {res.Status}");
+
+                if (res.Status == 200)
+                {
+                    dynamic dymamicBody = res.Body;
+
+                    foreach (var document in dymamicBody.documents)
+                    {
+                        Console.WriteLine($"id: {document.id} name: {document.detectedLanguage.name}");
+                    }
+                }
+            }
+
+            // Using the request builder pattern and the "builder" methods on JsonData.
+            {
+                DynamicRequest req = client.GetLanguagesRequest();
+
+
+                var body = new JsonData();
+                JsonData documents = req.Body.SetEmpty("documents", isArray: true);
```

Comment:
I wonder if we should not simply have two methods: SetEmptyArray and SetEmptyObject or even SetPropertyToEmptyArray/Object. Without this I worry that people will think the method sets the Body to empty

Reply:
Went with `SetEmptyArray` and `SetEmptyObject` for now.